Vasili Makouchyk/1/database.rb

```diff
@@ -0,0 +1,39 @@
+class DataBase
+  require 'nokogiri'
+  require 'mechanize'
+  require 'redis'
+  require 'open-uri'
+
+  URL = 'http://rapstyle.su/quotes.php'.freeze
+  FILE = 'citaty.html'.freeze
+
+  def initialize
+    @database = Redis.new
+    @quotes = []
+    download_page
+    scrape_quotes
+    set_db
+  end
+
+  def download_page
+    agent = Mechanize.new
+    page = agent.get(URL)
+    File.write(FILE, page.body)
+  end
+
+  def scrape_quotes
+    page = Nokogiri::HTML(File.open(FILE))
+    @quotes = page.css('.post-entry p')
+  end
+
+  def set_db
+    abort("Quotes hadn't been scraped") if @quotes.empty?
```

Comment:
You almost never need abort (at least in ruby).
https://en.wikipedia.org/wiki/Abort_(computing)
Why not use raise and let anyone calling your code handle the situation.
Also empty quotes doesn't seem like something that should be taken care of internally.
tl;dr use raise or remove the line